test/data/upgrade/upgrade_verify.sh

```diff
@@ -67,6 +69,29 @@ tee /etc/osbuild-composer/repositories/rhel-9.5.json > /dev/null << EOF
     ]
 }
 EOF
+else
```

Comment:
Same thing here, check for 10. Also not sure about being specific about 9.5 in the above block, which will need to get bumped with every new release won't it?

Reply:
Yes, but we always get a ticket to test a specific upgrade path so I'd like to be explicit here. If I try to get the latest compose or rely on any other automation I'm not sure I could assure that we test what's expected.